src/mmw/apps/modeling/views.py

```diff
@@ -235,8 +237,17 @@ def choose_worker():
     def predicate(worker_name):
         return settings.STACK_COLOR in worker_name or 'debug' in worker_name
 
+    @retry(Exception, delay=0.5, backoff=2, tries=3)
```

Comment:
I agree - let's see the extent to which we still experience problems before tweaking too much.